python/cuml/test/dask/test_random_forest.py

```diff
@@ -68,34 +70,47 @@ def _prep_training_data(c, X_train, y_train, partitions_per_worker):
 
 
 @pytest.mark.parametrize('partitions_per_worker', [3])
-def test_rf_classification_dask_cudf(partitions_per_worker, client):
+def test_rf_classification_multi_class(partitions_per_worker, cluster):
```

Comment:
Is this just the diff being shown weirdly, or did this multiclass test replace the binary test? I think we should have both tests included.

Reply:
It is just showing it weirdly, the updates replace the rf cpu multi classification test, we still have the binary classification test in the file.

Reply:
Ugh, diff fun! Thanks for clarifying